### PR TITLE
Editor Appearance demo: rework to show new DateRangeBox component

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by marker-dao — DISMISSED** [review]

File: JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js [hunk 1]
```diff
@@ -1,229 +1,176 @@
...
-          </div>
-        </div>
-        <div className="options">
-          <div className="caption">Options</div>
+export default function App() {
+  const defaultStylingMode = 'outlined';
+  const [stylingMode, setStylingMode] = React.useState(defaultStylingMode);
+  const [labelMode, setLabelMode] = React.useState('static');
+  const changeStylingMode = React.useCallback(({ value }) => {
+    setStylingMode(value);
```
Comment (ksercs) [inline JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js:37]: We have an agreement to add a setter as a dependency. Although it's meaningless from technical point, it's can help some users avoid similar errors
```suggestion
    setStylingMode(value);
```

File: JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js [hunk 2]
```diff
@@ -1,229 +1,176 @@
...
+  return (
+    <React.Fragment>
+      <div className="options">
+        <div className="caption">Options</div>
+        <div className="editors-modes">
           <div className="option">
             <label>Styling Mode</label>
             <SelectBox
-              stylingMode={this.state.stylingMode}
+              stylingMode={defaultStylingMode}
```
Comment (ksercs) [inline JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js:51]: hm, are u sure we have a "default" prefix for the `stylingMode`?

**Review by ksercs — DISMISSED** [review]

File: JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js (hunk 2 above)
Comment (alexanderPolosatov) [reply JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js:51]: it's my variable) we can rename to 'stylingModeDefaultValue'

File: JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js (hunk 1 above)
Comment (alexanderPolosatov) [reply JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js:37]: fixed

**Review by marker-dao — APPROVED** [review]

File: JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js (hunk 2 above)
Comment (ksercs) [reply JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js:51]: oh, missnote

**Review by ksercs — APPROVED** [review]

**Pull request merged**